wa/workloads/googleslides/__init__.py

```diff
@@ -121,3 +121,19 @@ def init_resources(self, context):
         # Only accept certain presentation formats
         if os.path.splitext(self.test_file.lower())[1] not in ['.pptx']:
             raise ValidationError('{} must be a PPTX file'.format(self.test_file))
+
+    def deploy_assets(self, context):
+        super(GoogleSlides, self).deploy_assets(context)
+        #Move the test files to the download folder
+        d = self.target.working_directory
+        e = self.target.external_storage
+
+        orig_file_path = self.target.path.join(d, self.test_file)
+        new_file_path = self.target.path.join(e, 'Download')
```

Comment:
Need to add the `new_file_path` to `self.deployed_assets` for the clean up to work correctly.

Reply:
Isn't this happening on line 133? 

self.deployed_assets.append(new_file_path)